packages/core/logger/src/Logger.js

```diff
@@ -2,27 +2,18 @@
 
 import type {IDisposable, LogEvent} from '@parcel/types';
 
-import EventEmitter from 'events';
+import {ValueEmitter} from '@parcel/events';
 import {inspect} from 'util';
 
 class Logger {
-  // TODO: This can't be explicitly annotated as an EventEmitter since
-  // declared private properties with type annotations break eslint's
-  // no-unused-var rule (even with babel-eslint). Annotate this when
-  // things aren't broken: https://github.com/babel/babel-eslint/issues/688
-  #emitter = new EventEmitter();
+  _logEmitter: ValueEmitter<LogEvent> = new ValueEmitter();
```

Comment:
moved this to an underscored property since this should be annotated, but the bug above prevents using a private property.